agent/consul/acl_server.go

```diff
@@ -217,10 +218,12 @@ func (s *Server) ResolveTokenToIdentityAndAuthorizer(token string) (structs.ACLI
 	return s.acls.ResolveTokenToIdentityAndAuthorizer(token)
 }
 
-func (s *Server) ResolveTokenAndDefaultMeta(token string, entMeta *structs.EnterpriseMeta, authzContext *acl.AuthorizerContext) (acl.Authorizer, error) {
+// ResolveTokenIdentityAndDefaultMeta retrieves an identity and authorizer for the caller,
```

Comment:
I added `ResolveTokenIdentityAndDefaultMeta` as an exact duplicate of `ResolveTokenAndDefaultMeta`, seeing that `s.acls.ResolveTokenToIdentityAndAuthorizer` gave me exactly what I needed, and the error handling was already in place. So, I decided to copy the original impl of `ResolveTokenAndDefaultMeta` and pass through to the new fn. However, this couples them. It seems fine from a de-duplicating functionality perspective, but I'm curious what the more Go-like choice here would be... I have a feeling that duplicating them would actually be preferred?

Reply:
It seems to be a pretty common practice even in Go to have wrap a method with another to get rid of some return values or to provide more/less parameters. So from that perspective this seems fine.